src/olympia/addons/serializers.py

```diff
@@ -644,6 +653,11 @@ def fake_object(self, data):
             for data_author in data_authors
         ]
 
+        if 'is_recommended' in data:
+            # if we have this property in the index, store it so we can expose
+            # it in get_is_recommended.
+            obj._is_recommended = data['is_recommended']
```

Comment:
Do we have a follow-up issue to remove this (and the code using it above) later ?

Reply:
https://github.com/mozilla/addons-server/issues/15271